### ISSUE TITLE
Question about the choice of dropout layer rate.

### ISSUE THREAD

**Issue by zhjuly** [titu1994/LSTM-FCN#10]
HI,
I have a small question. In LSTM model, you set the dropout rate 0.8, is it too large for LSTM to fit the data? I always set the rate 0.1~0.2. So, I would like to know the reason about the choice of it. Thanks a lot.

**Comment by titu1994** [comment]
The UCR dataset contains many easy datasets, and we train our models for nearly 2000 epochs, which is why to prevent overfitting from occuring too quickly, we use a very high dropout rate.
